server/db/migrate/20151216174050_add_quantity_to_add.rb

```diff
@@ -0,0 +1,7 @@
+class AddQuantityToAdd < ActiveRecord::Migration
+  def change
+
+    add_column :fusor_subscriptions, :quantity_to_add, :integer, :default => 0
```

Comment:
@jmrodri, I also defaulted `quantity_to_add` to be 0. At first I thought this was the issue, but I don't think so.